Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ApplyPaalmanPingsCorrection.py

```diff
@@ -49,8 +55,16 @@ def PyInit(self):
         self.declareProperty(name='CanShiftFactor', defaultValue=0.0,
                              doc='Amount by which to shift the container data')
 
-        self.declareProperty(MatrixWorkspaceProperty('OutputWorkspace', '', direction=Direction.Output),
+        self.declareProperty(MatrixWorkspaceProperty('OutputWorkspace', '',
+                             direction=Direction.Output),
                              doc='The output corrections workspace.')
+        self.declareProperty(name='RebinCanToSample',
+                             defaultValue='Rebin Can',
+                             validator=StringListValidator([
+                                 'Rebin Can',
+                                 'Do Not Rebin Can']),
+                             doc=('Enable or disable RebinToWorkspace on ' +
+                                  'CanWorkspace.'))
```

Comment:
Could be, but I rather use string selectors. It forces self-documenting code:
```python
ApplyPaalmanPingsCorrection('samplews', 'corrws', 'canws', 1.0, 0.0, 'outws', 'Rebin Can')
```
is easier to read than
```python
ApplyPaalmanPingsCorrection('samplews', 'corrws', 'canws', 1.0, 0.0, 'outws', True)
```
What is `True` here?